examples/example-menubar/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@lumino/example-menubar",
+  "version": "0.9.0-alpha.6",
+  "private": true,
+  "scripts": {
+    "build": "tsc && rollup -c",
+    "clean": "rimraf build"
+  },
+  "dependencies": {
+    "@lumino/default-theme": "^1.0.0-alpha.6",
+    "@lumino/messaging": "^2.0.0-alpha.6",
+    "@lumino/widgets": "^2.0.0-alpha.6"
```

Comment:
I realized I'm actually quite confused about how this builds the example with your local modifications to the source code. I think it doesn't do so automatically, so in order to test this new example page with your source code changes, I think I have to execute the following commands:

```shell
cd lumino
gh pr checkout 465
cd packages/widgets
yarn run build
yarn link
cd ../../examples/example-menubar
yarn install
yarn link @lumino/widgets
yarn run build
open index.html
```

Reply:
I hadn't even thought about that. I just run

```
cd packages/widgets
yarn run build
cd ../../examples/example-menubar
yarn run build
```

and it works :-).